telegram/bot.py

```diff
@@ -3768,9 +3950,17 @@ def __reduce__(self):
     """Alias for :attr:`set_sticker_position_in_set`"""
     deleteStickerFromSet = delete_sticker_from_set
     """Alias for :attr:`delete_sticker_from_set`"""
+    setStickerSetThumb = set_sticker_set_thumb
+    """Alias for :attr:`set_sticker_set_thumb`"""
     setPassportDataErrors = set_passport_data_errors
     """Alias for :attr:`set_passport_data_errors`"""
     sendPoll = send_poll
     """Alias for :attr:`send_poll`"""
     stopPoll = stop_poll
     """Alias for :attr:`stop_poll`"""
+    sendDice = send_dice
+    """Alias for :attr:`send_dice`"""
+    getMyCommands = get_my_commands
+    """Alias for :attr:`get_my_commands`"""
+    setMyCommands = set_my_commands
+    """Alias for :attr:`set_my_commands`"""
```

Comment:
For some reason I remembered that we deprecated the camelCase aliases 🤔

Oh well...

Reply:
Not that I know of. Maybe for v13? We only need to adjust `test_official`, but that should be doable

Reply:
Doesn't really matter except I'm hallucinating